src/apps/statements/tests/storingStatements/queriables/getMetadataFromStatement/getMetadataFromStatement.test.ts

```diff
@@ -0,0 +1,50 @@
+import * as assert from 'assert';
+import { merge } from 'lodash';
+
+import getMetadataFromStatement
+  from '../../../../service/storeStatements/queriables/getMetadataFromStatement';
+import {
+  sequencingInteractionActivityStatement,
+  statementDefaults,
+} from './fixtures/statements.fixture';
+
+describe('Retrieve metadata from statement', () => {
+  it('should retrieve duration metadata from statement', () => {
+    const actualDurationMetadata = getMetadataFromStatement(statementDefaults);
+    const expectedDurationMetadata = {
+      'https://learninglocker&46;net/result-duration': { seconds: 37080306 },
+    };
+
+    assert.deepEqual(actualDurationMetadata, expectedDurationMetadata);
+  });
+
+  it('should return sequencing metadata from statement', () => {
+    const actualSequencingMetadata = getMetadataFromStatement(
+      sequencingInteractionActivityStatement,
+    );
+    const expectedSequencingMetadata = {
+      'https://learninglocker&46;net/sequencing-response': ['tim', 'mike', 'ells', 'ben'],
+    };
+
+    assert.deepEqual(actualSequencingMetadata, expectedSequencingMetadata);
+  });
+
+  it('should return duration and sequencing metadata from one statement', () => {
+    const statementWithDurationAndSequencing = merge(
```

Comment:
I think we could have used the spread operator here, but this test is exactly what was needed.